fladle-plugin/src/main/java/com/osacky/flank/gradle/FulladlePlugin.kt

```diff
@@ -24,115 +23,127 @@ class FulladlePlugin : Plugin<Project> {
     }
 
     val fulladleConfigureTask = root.tasks.register("configureFulladle") {
+      var allModulesDisabled = true
```

Comment:
I think it is easier to reason about the logic when variables have positive names. Can we rename it like so?
```suggestion
      var modulesConfigured = false
```